apps/ashop/src/helpers/cruiseInfo.ts

```diff
@@ -49,25 +49,30 @@ export async function getTrips(appMode: string) {
         const wcgopTrips = getDocByType(appMode, 'trip');
         return wcgopTrips;
     }
-
 }
 
-// TODO
-// 1. call cruise API to generate friendly cruiseNum
-// 2. Find a way to indicate which cruise is active on the userDB. Either remove
-// all inactive data each time a cruise ends. Or add a column that
-// indicates whether cruise is active.
-export function updateCruise(tripId: CouchID) {
-    if (cruise) {
-        cruise.trips ? cruise.trips.push(tripId) : (cruise.trips = [tripId]);
-        db.put(cruise);
-    } else {
-        const newCruise = { type: 'ashop-cruise', trips: [tripId] };
-        db.post(newCruise);
+export async function createCruise() {
+    const cruises = await getDocByType('ashop', 'cruise');
+    if (cruises && cruises.length === 0) {
+        const cruiseNum = 1; // TODO use API to generate cruiseId
```

Comment:
Maybe default to 10001 or something large, so it'll be easy to see if this has been updated from the Cruise API